src/components/opt-in-out/index.tsx

```diff
@@ -3,7 +3,6 @@ import { useRouter } from 'next/router'
 import dynamic from 'next/dynamic'
 import Cookies from 'js-cookie'
 import { IconSignOut16 } from '@hashicorp/flight-icons/svg-react/sign-out-16'
-import '@reach/dialog/styles.css'
```

Comment:
Moved this to the dialog component.[ See related issue.](https://github.com/hashicorp/dev-portal/pull/837#issuecomment-1218564056)